### ISSUE TITLE
no points found

### ISSUE THREAD

**Issue by robinscheurwater** [matthieuheitz/google-data-analyses#11]
Google renamed the json file to Records.json.

Also the timestamp key is renamed. Therefore the total number of points keeps returning zero.

**Comment by delagra** [comment]
Yes, I had to rewrite the cycle that creates the timestampMs list. Timestamp is no longer in milliseconds in the json file so it has to be converted

**Comment by YanisKyr** [comment]
@delagra care sharing your updated code?

**Comment by Highlanderix** [comment]
> Yes, I had to rewrite the cycle that creates the timestampMs list. Timestamp is no longer in milliseconds in the json file so it has to be converted

Hello, I would be great if you can share that block of code with the rest of us. I had already found that this was the issue, but as I'm not familiar with Python, I didn't manage to get it working.

Thanks

**Comment by dwaycik** [comment]
> Yes, I had to rewrite the cycle that creates the timestampMs list. Timestamp is no longer in milliseconds in the json file so it has to be converted

@delagra Would greatly appreciate some insight on how to re-write this... I tried removing milliseconds; no dice.

I'm hesitant to start combing through the JSON cause it's so big LOL... thanks in advance if you don't mind sharing this :)

**Comment by johnfoxii** [comment]
The file definition for Records.json seems to be in question. https://locationhistoryformat.com/reference/records/#/$defs/locationRecord/
The author may have meant to say he is working on it.

**Comment by Highlanderix** [comment]
Hello, after some tinkering with Python for something else, I give this a try and managed to get it working : https://github.com/matthieuheitz/google-data-analyses/pull/13 . Enjoy !
